### PR TITLE
Create A route for both Home page and About 

### PR DESCRIPTION
**Pull request by Yjohn**

### CODE REVIEW

File: client/src/containers/app/App.test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from 'containers/App';
```
Comment (kabaros) [inline client/src/containers/app/App.test.js]: why did you change this?

**Comment by kabaros** [comment]
delete the node_modules in the root folder and commit. And fix the note about the test. @Yjohn

File: client/src/containers/app/App.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
 import React, { Component } from 'react';
-import logo from '../logo.svg';
 import './App.css';
+import About from '../../components/About/About';
+import Home from '../../components/Home/Home';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```
Comment (kabaros) [inline client/src/containers/app/App.js]: shouldn't you be using react-router - not react-router-dom

also use **save** so that package.json is updated `npm install react-router --save`

**Comment by kabaros** [comment]
also can you run `npm run test` and try to fix the fail unit test (App.test.js)

File: client/src/containers/app/App.js (hunk 2 above)
Comment (kabaros) [reply client/src/containers/app/App.js]: @Yjohn did you see this comment? ^

Comment (Yjohn) [reply client/src/containers/app/App.js]: yeah, I see
 ```PASS  src/containers/App/App.test.js
  ✓ renders without crashing (10ms)
Test Suites: 1 passed, 1 totalTests:       1 passed, 1 totalSnapshots:   0 total
Time:        0.203s
Ran all test suites.```

Comment (Yjohn) [reply client/src/containers/app/App.js]: without `react-router-dom` it doesn't work the `route`

**Comment by kabaros** [comment]
Ok could you just use npm --save so that is added in package.json

sent from my phone

**Comment by kabaros** [comment]
@Yjohn why did you close this PR??

**Comment by kabaros** [comment]
all you need is to react-router-dom to package.json (i.e. use npm install --save

**Comment by Yjohn** [comment]
cause i hate it 😡😡

**Comment by kabaros** [comment]
add react-router-dom to package.json and you're done
